packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -93,6 +93,17 @@ export class KeybindingWidget extends ReactWidget {
         this.focusInputField();
     }
 
+    // Calculate the net rendered result
```

Comment:
Can we update the documentation to be consistent with out [coding guidelines](https://github.com/theia-ide/theia/wiki/Coding-Guidelines#comments).
We use [JSDoc](https://en.wikipedia.org/wiki/JSDoc) style documentation which you can see an example of here:

https://github.com/theia-ide/theia/blob/798ecc9304f395103e804d705a86565c14507364/packages/cpp/src/browser/cpp-build-configurations.ts#L74-L79